### PR TITLE
Changed Assignment Request Handling

### PR DESCRIPTION
**Pull request by OmSingh5092**
### Updates

1- **Updated Assignment Relational Model** 
       Added few necessary fields like _**assignment_title, assignment_description, image_path**_

2- **Configured Cloudinary to handle Image upload request**

3- **Added Post request to handle image uploads to Cloudinary**
Added _multer_ and _uriparser_ to handle formdata

### Screen Shots
**Assignment get request response.**

image

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -26,20 +26,18 @@ var db = {
...
     },
-
-
 };
 
 var cache = {
     rg: {
         host: 'localhost',
-        port: 6379
+        port: 5433
```
Comment (antailbaxt3r) [inline config/config.js:40]: Revert changes in this file

File: config/config.js [hunk 2]
```diff
@@ -54,7 +52,7 @@ var config = {
     },
 
     db: {
-        env: db.staging, // This is where we set the environment of the db
+        env: db.rg, // This is where we set the environment of the db
```
Comment (antailbaxt3r) [inline config/config.js:55]: Revert

File: models/db.js [hunk 3]
```diff
@@ -24,4 +25,12 @@ db.public.assignments.belongsTo(db.public.login, { foreignKey: 'user_id', onDele
...
 db.public.classes.belongsTo(db.public.login, { foreignKey: 'login_id', onDelete: 'CASCADE', constraints: false, allowNull: false });
 db.public.classes.belongsTo(db.public.subjects, { foreignKey: 'subject_id', onDelete: 'CASCADE', constraints: false, allowNull: false });
+
+//For making the relations on the local database.
+/*sequelize.sync({ force: true })
+  .then(() => {
+    console.log(`Database & tables created!`)
+  })  
+*/
+
```
Comment (antailbaxt3r) [inline models/db.js:35]: Unnecessary comment

File: models/public/db.js [hunk 4]
```diff
@@ -11,15 +11,16 @@ const sequelize = new Sequelize(env.DATABASE_NAME, env.DATABASE_USERNAME, env.DA
...
     schema: env.SCHEMA
   },
-  dialectOptions: {
+  //Removed to test local database without ssl
+  /*dialectOptions: {
     ssl: {
-      require: true,
-      rejectUnauthorized: false
+      require: false,
+      rejectUnauthorized: true
```
Comment (antailbaxt3r) [inline models/public/db.js:18]: Don't remove ssl verification

File: routes/assignments.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
 
 var verify = require('../functions/verifyFunc');
 var validate = require('../functions/validateFunc');
 
 var assignments = require('../controllers/assignmentsCtrl')
 
-router.get('/get', verify.user, assignments.getAssignments);
-router.post('/add', verify.user, validate.assignmentValidation, assignments.createAssignment)
+router.get('/get', verify.user,assignments.getAssignments);
+router.post('/add', verify.user, validate.assignmentValidation ,assignments.createAssignment)
```
Comment (antailbaxt3r) [inline routes/assignments.js:12]: Unnecessary change

File: models/public/db.js (hunk 4 above)
Comment (OmSingh5092) [reply models/public/db.js:18]: Removed for using local server without SSL verification.

File: models/db.js (hunk 3 above)
Comment (OmSingh5092) [reply models/db.js:35]: called this function to initialize tables in Postgres
